src/BlockElement/AbstractStaticSelect.php

```diff
@@ -2,7 +2,7 @@
 namespace Maknz\Slack\BlockElement;
 
 use InvalidArgumentException;
-use Maknz\Slack\Object\OptionGroup;
+use Maknz\Slack\CompositionObject\OptionGroup;
```

Comment:
this breaks BC, for ex., if some one already extends these classes.
so, It's better to keep the old name.

Reply:
mmm... sorry, I already forget all of things.
Do you try to add php8.4 support to `2.x` branch ?
Is it possible ?

Reply:
There seem to be some extraneous commits in here. Thought I branched off main. Will check.

Reply:
I did branch from master, so this was already there. Not sure why, but you should probably fix, as it is not my change.